models/frax_finance/bnb/frax_finance_bnb_bribes.sql

```diff
@@ -0,0 +1,465 @@
+{{ config(
+    alias = 'bribes'
+    )
+}}
+
+
+with base_date as (
+    WITH t1 AS (
+        SELECT
+            explode(
+                sequence(
+                    to_date('2023-01-05'),
+                    current_date,
+                    interval 7 days
+                )
+            ) AS week_array
+    )
+    SELECT
+        week_array AS week_start,
+        weekofyear(date_sub(week_array, 4)) AS week,
+        date_add(week_array, 7) AS week_end
+    FROM
+        t1
+),
+pairs as (
+    select
+        distinct contract_address,
+        output_0 as token0,
+        output_1 as token1
+    from
+        { { source('thena_fi_bnb', 'Pair_call_tokens') } }
```

Comment:
Can you rewrite all `{ {` into `{{`, it seems the current { { causes [the compilation error](https://github.com/duneanalytics/spellbook/actions/runs/5146831224/jobs/9266427509?pr=3258#step:9:55).

```suggestion
        {{ source('thena_fi_bnb', 'Pair_call_tokens') }}
```

Reply:
@Hosuke  Done!